### PR TITLE
Toggle ranks

### PR DESCRIPTION
**Pull request by dolivas644**
Issue: Top Ranking page keeps breaking

### CODE REVIEW

**Comment by dolivas644** [comment]
Issue: Review crads not displaying 

- ReviewCard is mapped it is the only json that has a nested data structure {data: data{...All my info that I need is here }}
- fetch request is made and updates selected state
- request is called on click
- header appears notifying state is changing BUT cards do NOT appear

**Comment by dolivas644** [comment]
Addition: Also need to add a default state to display anime content 
const [selected, setSelected] = useState('anime') 
Didnt change the default to anime

**Comment by cdm-ium** [comment]
As a first step to fixing some of your bugs, you should read through the errors you're getting in the console and try to fix all the ones you can and understand why any are happening that you choose not to fix. Doing so will probably fix some of your bugs. If you think an error is related to one of your bugs, but you can't find the fix, you should include the errors in your pr.

Screen Shot 2022-11-13 at 22 00 43

**Comment by cdm-ium** [comment]
It's more helpful if you put your comments in teh PR on specific lines of the code. You can't start discussion threads for comments at the top level of the pr.

**Comment by cdm-ium** [comment]
I believe I see the issue with your topreviews page. Some hint/questions:

1. What have you tried to debug this?
2. What data do you think should be displayed--be as explicit as possible--that is not displayed?

**Comment by cdm-ium** [comment]
Overall what makes you say the TopRanking page keeps breaking? I just tried it out locally. Except for the topreviews being empty, everything else looks pretty good.

File: client/src/components/Ranks/TopRanks.js [hunk 1]
```diff
@@ -1,14 +1,111 @@
 import React from 'react'
-
+import { useState , useEffect} from 'react';
+import AnimeCard from './cards/AnimeCard';
```
Comment (cdm-ium) [inline client/src/components/Ranks/TopRanks.js:3]: I appreciate you split this out this way, with one main container card delegating to card components of different types. This is way clearer and I understand what you were doing immediately. You did a good job with writing this part. Making it so other people can understand what you're doing is a big part of the battle!

**Pull request merged**